lib/cloak/padding.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Cloak.Padding do
+  def pad(str, size \\ 16) do
+    if byte_size(str) < size do
+      str
+      |> Kernel.<>("\x80")
+      |> String.pad_trailing(size - 1, "\x00")
+    else
+      str
```

Comment:
Also, we don't have any test coverage of our line. We should have a test that tests what happens if the string is already longer than the padding.